src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -688,18 +692,20 @@ private Goal parseGoal(String[] params) {
         Goal goal;
         try {
             BetterBlockPos playerFeet = ctx.playerFeet();
-            switch (params.length) {
+
+            // tests wether a dimension input exists because the switch has to be handled differently
+            switch (params.length < 1 || (isNumeric(params[params.length - 1]) || params[params.length - 1].startsWith("~")) ? params.length : params.length - 1) {
```

Comment:
disgusting

Reply:
split this out into a length before switch instead of a gross ternary.

```
int length;
if (params.length < 1 || (isNumeric(params[params.length - 1]) || params[params.length - 1].startsWith("~")) {
   length = params.length;
}else{
   length  = params.length - 1;
}
switch(length){
```

Reply:
sure boss, I also like to have it split up instead of 1 line